Enhance product details documentation with scoped contexts support (#432)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/product-details/containers/product-attributes.mdx

import OptionsTable from '@components/OptionsTable.astro';

The `ProductAttributes` container displays a list of attributes for a product on the product details page.

> The container receives initial product data to preload the component during [initialization](/dropins/product-details/initialization/), and it's event-driven so it gets updated with data emitted to `pdp/data` in the scope of the event.

Reviewer: ```suggestion
The container receives initial product data during [initialization](/dropins/product-details/initialization/) to preload the component and, being event-driven, updates with data emitted to `pdp/data` within the event scope.
```
